### ISSUE TITLE
How to define a non-virtual getter

### ISSUE THREAD

**Issue by bangbang93** [szokodiakos/typegoose#192]
If using schema, I can do like this
``` typescript
{
    type: String,
    index: 'text',
    get(this: IMessageSchema, message) {
      if (this.type === Message.EnumType.deleted) {
        return ''
      }
      return message
    },
  },
```
But it seems like if i using get in typegoose, it must be a virtual field

**Comment by hasezoey** [comment]
in typegoose you can only do virtual properties with get- & setters
-> can it be done otherwise in mongoose? because currently it is only a pass-through, so when mongoose supports it, it should be possible

(i know this is old)

**Comment by hasezoey** [comment]
this can be closed in favor of #260 
-> included in 6.0.0 (since 6.0.0-9)

@Ben305

**Comment by hasezoey** [comment]
@Ben305 Bump to close
